### PR TITLE
Kk/theme engine

### PR DESCRIPTION
**Pull request by kautukkundan**
## What is this PR doing?
- Adds tailwind css config
- replaces styling of WelcomeScreen component to use tailwind

## How can these changes be manually tested?
- build and import extension into chrome 

## Checklist
- [x] I have manually tested these changes
- [ ] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: extension/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+module.exports = {
```
Comment (kautukkundan) [inline extension/tailwind.config.js:1]: Added only required config for current scope. Will add more preset styles based on the Quill Design system as required.

File: extension/source/components/Button.tsx [hunk 2]
```diff
@@ -1,52 +1,26 @@
+import { IconProps } from 'phosphor-react';
 import * as React from 'react';
 
 const Button = (props: {
+  className?: string;
```
Comment (kautukkundan) [inline extension/source/components/Button.tsx:5]: I realized that the same button component will be reused with variation at different places so I moved the styling outside such that it is implemented in parent where button is used.

File: extension/source/components/Button.tsx [hunk 3]
```diff
@@ -1,52 +1,26 @@
...
   onPress: () => void;
-  highlight?: boolean;
+  highlight?: boolean; //TODO - remove dependency
```
Comment (jacque006) [inline extension/source/components/Button.tsx:7]: What dependency needs to be removed for this? Would be nice to document in this comment.

**Review by jacque006 — APPROVED** [review]
:+1: Nice simple start

File: extension/source/components/Button.tsx (hunk 3 above)
Comment (voltrevo) [reply extension/source/components/Button.tsx:7]: https://github.com/jzaki/bls-wallet/issues/110

**Review by voltrevo — APPROVED** [review]

File: extension/source/components/Button.tsx (hunk 3 above)
Comment (kautukkundan) [reply extension/source/components/Button.tsx:7]: added issue description 👍🏼

**Pull request merged**